packages/cli/src/index.js

```diff
@@ -83,14 +84,58 @@ const loadDotEnvDefaultsMiddleware = () => {
   })
 }
 
+const updateCheckerMiddleware = (argv) => {
+  const excludedCommands = ['update', 'upgrade', 'ts-to-js']
+  if (excludedCommands.includes(argv._[0])) {
+    return
+  }
+  if (updateCommand.shouldShowUpgradeAvailableMessage()) {
+    process.on('exit', () => {
+      updateCommand.showUpgradeAvailableMessage()
+    })
+  }
+  if (updateCommand.isUpdateCheckDue()) {
+    const backgroundUpdateBaseLogPath = path.join(
+      getPaths().base || '/tmp',
+      '.redwood'
+    )
+    const out = fs.openSync(
+      path.join(
+        backgroundUpdateBaseLogPath,
+        'background-update-checker.out.log'
+      ),
+      'a'
+    )
+    const err = fs.openSync(
+      path.join(
+        backgroundUpdateBaseLogPath,
+        'background-update-checker.err.log'
+      ),
+      'a'
+    )
+    const child = spawn('yarn', ['rw', 'update', '--silent'], {
+      detached: true,
+      stdio: ['ignore', out, err],
+      cwd: getPaths().base,
+      env: { ...process.env },
+      shell: process.platform === 'win32',
+    })
+    child.unref()
+  }
+}
+
 // eslint-disable-next-line no-unused-expressions
 yargs
   .scriptName('rw')
-  .middleware([
-    getCwdMiddleware,
-    loadDotEnvDefaultsMiddleware,
-    telemetryMiddleware,
-  ])
+  .middleware(
+    [
+      getCwdMiddleware,
+      loadDotEnvDefaultsMiddleware,
+      telemetryMiddleware,
+      !process.env.REDWOOD_BACKGROUND_UPDATES_DISABLED &&
```

Comment:
Unfortunately I'm not sure if we have a solid convention for env vars, but I would want to make sure this resembles the others, whether everything's prefixed with `REDWOOD` or `RW`, etc. (Anyone know?) I know for contributing, there's `RWFW_PATH`, and even `RW_PATH` I think. But that's a bit different from app env vars.

We could also read from the TOML for this

Reply:
Prisma calls their env var `PRISMA_HIDE_UPDATE_MESSAGE`. Maybe something like that reads a little easier?